examples/c++/stablehlo_add.cpp

```diff
@@ -0,0 +1,54 @@
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/Quant/QuantOps.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/Verifier.h"
+#include "stablehlo/dialect/StablehloOps.h"
+
+int main() {
+  mlir::MLIRContext context;
+
+  /** create module **/
+  mlir::OwningOpRef<mlir::ModuleOp> module =
+      mlir::ModuleOp::create(mlir::UnknownLoc::get(&context));
+  module->getContext()->loadDialect<mlir::func::FuncDialect>();
+  module->getContext()->loadDialect<mlir::stablehlo::StablehloDialect>();
+  module->getContext()->loadDialect<mlir::quant::QuantizationDialect>();
+  module->setName("test_module");
+
+  /** create function **/
+  // create function argument and result types
+  mlir::Type arg0 =
+      mlir::RankedTensorType::get({3, 4}, mlir::FloatType::getF32(&context));
+  auto func_type = mlir::FunctionType::get(&context, {arg0, arg0}, {arg0});
+
+  // create the function and map arguments.
+  llvm::ArrayRef<mlir::NamedAttribute> attrs;
+  auto function = mlir::func::FuncOp::create(mlir::UnknownLoc::get(&context),
+                                             "main", func_type, attrs);
+  function.setVisibility(mlir::func::FuncOp::Visibility::Public);
+  module->push_back(function);
+
+  // create function block with add operations
+  mlir::Block* block = function.addEntryBlock();
+  llvm::SmallVector<mlir::Value, 4> arguments(block->args_begin(),
+                                              block->args_end());
+  mlir::OpBuilder block_builder = mlir::OpBuilder::atBlockEnd(block);
+  mlir::Location loc = block_builder.getUnknownLoc();
+
+  llvm::SmallVector<mlir::NamedAttribute, 10> attributes;
+  block_builder.create<mlir::stablehlo::AddOp>(loc, arguments, attributes)
+      .getOperation();
+
+  mlir::Operation* op =
+      block_builder
+          .create<mlir::stablehlo::AddOp>(loc, arg0, arguments, attributes)
+          .getOperation();
+  block_builder.create<mlir::func::ReturnOp>(loc, op->getResult(0));
+
+  // verify the module and dump
```

Comment:
For consistency lets use `/**/` comment style we were using for high level actions?